jsonrpcserver/async_dispatcher.py

```diff
@@ -30,6 +31,8 @@
 from .utils import make_list
 
 
+logger = logging.getLogger("jsonrpcserver")
```

Comment:
Better to use `__name__`?

Reply:
Within the scope of this project it wouldn't make a difference.

Reply:
Agree but there may be more loggers added in the future (unlikely) but `__name__` would allow you to configure the loggers for each module individually

Reply:
Understood. Updated with __name__.